src/@orb.yml

```diff
@@ -25,13 +25,18 @@ examples:
           jobs:
             - ggshield/scan:
                 name: ggshield-scan  # best practice is to name each orb job
-                base_revision: <<pipeline.git.base_revision>>
-                revision: <<pipeline.git.revision>>
-
+                number_commits_gh: <<pipeline.trigger_parameters.github_app.total_commits_count>>
+                number_commits_gl: <<pipeline.trigger_parameters.gitlab.total_commits_count>>
```

Comment:
Would it work to remove the `number_commits_*` arguments from there and initialize our environment variables using`<<pipeline.trigger_parameters.*.total_commits_count>>` instead of `<<parameters.number_commits_*>>`?

Reply:
Good point, I was actually asking myself that, I'll try

Reply:
I see CircleCI supports BitBucket in addition to GitHub and GitLab. Should we add this one too? (Assuming the commit count is exposed similarly)

Reply:
Oh, too bad for BitBucket. We can't do much if the info is not there.